client/asset/btc/btc.go

```diff
@@ -1849,8 +1848,8 @@ func (btc *ExchangeWallet) sendWithReturn(baseTx *wire.MsgTx, addr btcutil.Addre
 	if err != nil {
 		return makeErr("signing error: %v, raw tx: %x", err, btc.wireBytes(baseTx))
 	}
-	size := msgTx.SerializeSize()
-	minFee := feeRate * uint64(size)
+	vSize := dexbtc.MsgTxVBytes(msgTx)
+	minFee := feeRate * vSize
```

Comment:
Part of me thinks this function should do feeRate++ to be safe. One extra sat/vbyte isn't going to be costly, but it also needs to be accounted for when funding an order.